### PR TITLE
add home page

### PR DESCRIPTION
**Pull request by kavyakunder**
Pomodoro Home Page :
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,40 @@
-import logo from './logo.svg';
-import './App.css';
+import Home from "./pages/Home/Home";
+// import Timer from "./pages/Timer/Timer";
```
Comment (kedark152) [inline src/App.js:2]: Suggestion: you can remove comments if not needed in future.

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,40 @@
...
+            <li className="list-item-inline">
+              <Link className="link" to="/timer">
+                Timer
+              </Link>
+            </li>
+          </ul>
+        </nav>
+
+        <Routes>
+          {/* <Route path="/timer" element={<Timer />} /> */}
```
Comment (kedark152) [inline src/App.js:32]: Suggestion: you can remove comments if not needed in future.

File: src/pages/Home/home.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
+body {
+  font-family: "Roboto", sans-serif;
```
Comment (kedark152) [inline src/pages/Home/home.css]: You can use variables for font-family.

File: src/pages/Home/home.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  text-decoration: none;
+  margin: 1rem;
+  padding: 3rem 1rem;
+  box-sizing: border-box;
+  font-size: 1.8rem;
+  color: black;
+}
+
+.link:hover {
+  color: #5d5d5d;
```
Comment (kedark152) [inline src/pages/Home/home.css]: you can use this color as variable so you can re-use it.

**Review by kedark152 — APPROVED** [review]
Great Design for pomodoro app, code looks good.
suggested few small things, you can think upon it,

File: src/pages/Home/Home.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./home.css";
+function Home() {
+  return (
+    <>
+      <body>
```
Comment (junaidshaikh-js) [inline src/pages/Home/Home.js]: You should not return body tag as there should be only one body tag per page

**Review by junaidshaikh-js — COMMENTED** [review]
Good work! 

Some changes can be done as suggested. Clean code is written.

**Pull request merged**